shared/README.md

```diff
@@ -364,6 +364,22 @@ See [this
 link](https://github.com/guard/listen/wiki/Increasing-the-amount-of-inotify-watchers)
 for how to increase the watch limit; I set mine to 65536.
 
+On Android 28 and above HTTP traffic is disabled by default which can block
+Metro Bundler from running properly. We have manually allowed `127.0.0.1` to
+have HTTP traffic, so if you see an error about connecting to the bundler
+server you should manually change the dev server URL and then kill and restart
+the app:
+
+```sh
+# Enable loopback
+adb reverse tcp:8081 tcp:8081
+```
+
+Then open the [react native debug
+menu](https://facebook.github.io/react-native/docs/debugging.html#accessing-the-in-app-developer-menu),
+tap "Dev settings" and set "Debug server host & port for device" to
+`127.0.0.1:8081`
```

Comment:
is the default not already `127.0.0.1:8081`?

Reply:
nojima said:
> It should be actually

so maybe the language should be
`...to 127.0.0.1:8081 (the default) if it isn't already`

Reply:
it wasn't the default for me but maybe something has been changed since i set things up